azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/__init__.py

```diff
@@ -23,31 +32,51 @@ def configure_azure_monitor(**kwargs):
     """
 
     configurations = get_configurations(**kwargs)
-    disable_tracing = configurations.get("disable_tracing", False)
+    connection_string = configurations.get("connection_string", False)
     service_name = configurations.get("service_name", "")
     service_namespace = configurations.get("service_namespace", "")
     service_instance_id = configurations.get("service_instance_id", "")
+    disable_logging = configurations.get("disable_logging", False)
+    logging_level = configurations.get("logging_level", NOTSET)
+    disable_tracing = configurations.get("disable_tracing", False)
     sampling_ratio = configurations.get("sampling_ratio", 1.0)
     tracing_export_interval_millis = configurations.get(
         "tracing_export_interval_millis", 30000
     )
 
-    if not disable_tracing:
+    resource = None
+    if not disable_logging or not disable_tracing:
         resource = Resource.create(
             {
                 ResourceAttributes.SERVICE_NAME: service_name,
                 ResourceAttributes.SERVICE_NAMESPACE: service_namespace,
                 ResourceAttributes.SERVICE_INSTANCE_ID: service_instance_id,
             }
         )
+    if not disable_logging:
+        logging_provider = LoggerProvider(resource=resource)
+        log_exporter = AzureMonitorLogExporter(
+            connection_string=connection_string
+        )
+        logging_provider.add_log_record_processor(
+            BatchLogRecordProcessor(log_exporter)
+        )
+        handler = LoggingHandler(
+            level=logging_level, logger_provider=logging_provider
+        )
+        getLogger().addHandler(handler)
```

Comment:
I don't think we can just add this to the root logger. We might need to have another configuration to specify the logger that we want to add the handler to. Can leave this as is but create a github tracking issue for this in the meantime.

Reply:
This is how init_logging does it. But we could make it configurable if we'd like

Reply:
init_loggin adding handler to root logger: https://github.com/open-telemetry/opentelemetry-python/blob/release/v1.14.x-0.35bx/opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py#L226

While we could allow users to chose their base logger, it would be an additional feature we would then have to support on top of OTel

Reply:
We shouldn't be comparing our functionality to OT's auto instrumentation functionality (because it could be not desirable for OUR azure customer use case). I'm okay with adding to root logger for now, but please create a tracking issue to implement the configurable feature.

Reply:
https://github.com/microsoft/ApplicationInsights-Python/issues/223